tests/test_frontend_parity_186.py

```
@@ -0,0 +1,130 @@
+        " input_extensions: [...m.inputExtensions].sort(),"
+        " supports_compression: !!m.supportsCompression,"
+        " supports_compression_level: !!m.supportsCompressionLevel,"
+        " supports_delete_on_verify: !!m.supportsDeleteOnVerify,"
+        " allows_archive_input: !!m.allowsArchiveInput,"
+        "})));\n"
+        "process.stdout.write(JSON.stringify(__rows));\n"
+    )
+    script = tmp_path / "registry_eval.mjs"
+    script.write_text(src, encoding="utf-8")
+
+    proc = subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)